src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form-field form-field--big"
+      data-qa="big"
+    >
```

Comment:
The `<form>` tag should not be self-closing. In HTML, the `<form>` element should have an opening and a closing tag. Please change `<form />` to `<form>` and ensure it has a corresponding `</form>` closing tag.